packages/block-library/src/classic/edit.js

```diff
@@ -216,3 +221,16 @@ export default class ClassicEdit extends Component {
 		/* eslint-enable jsx-a11y/no-static-element-interactions */
 	}
 }
+
+const LazyClassicEdit = ( props ) => (
+	<LazyLoad
+		scripts={ [ 'wp-tinymce' ] }
+		styles={ [ 'wp-tinymce' ] }
+		onLoaded={ () => window.wpMceTranslation() }
```

Comment:
I'm not actually sure whether this would be how we do it. Given we can fully control what gets printed when Gutenberg is loading through the `print_tinymce_scripts` we can 100% wrap `wp_mce_translation`'s result in a function to run in `onLoaded`... but it's also been floated that along with the new REST API that will provide the TinyMCE script, we could return the info necessary for initializing the library's i18n as a JSON blob.

Right now I _think_ I prefer just printing this wrapped in a function in the action, but curious what the trade offs people can think of would be.